packages/cxl-ui/src/components/cxl-dashboard-team-header.js

```diff
@@ -26,16 +28,16 @@ export class CxlDashboardTeamHeaderElement extends LitElement {
             <span class="subtitle">${this.multiple ? 'Choose team' : 'My team'}</span>
             ${this.multiple
               ? html`<slot name="choose-team"></slot>`
-              : html`<h1 class="title">${this.name}</h1>`}
+              : html`<h1 class="title">${this.teamName}</h1>`}
           </div>
         </header>
         <div class="actions">
-          <a>
+          <a href="my-account/teams/${this.teamId}/members/">
             <vaadin-button class="invite-manage" theme="secondary">
               Invite & manage team
             </vaadin-button>
           </a>
-          <a>
+          <a href="my-account/teams/${this.teamId}/settings/">
```

Comment:
These links should be passed by attribute. We need flexibility just in case URL changes.

Reply:
@freudFlintstone this is missing